### PR TITLE
Repositioned search bar

### PR DESCRIPTION
**Pull request by 4RHollingsworth**
Moved the search bar from the header to the home page

### CODE REVIEW

File: client/vue.config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = defineConfig({
   devServer: {
     proxy: {
       "^/api": {
-        target: "http://localhost:5000",
+        target: "http://localhost:5001",
```
Comment (zaidi-01) [inline client/vue.config.js:8]: Let's leave the default port as 5000.

File: client/src/router/index.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ const router = createRouter({
   routes,
 });
 
-router.beforeEach(async (to, from, next) => {
+/* router.beforeEach(async (to, from, next) => {
```
Comment (zaidi-01) [inline client/src/router/index.js:65]: I believe this is to debug, please undo it.

File: client/src/components/Home.vue [hunk 3]
```diff
@@ -2,6 +2,18 @@
   <div>
     <Header />
 
+    <div class="search-bar">
```
Comment (zaidi-01) [inline client/src/components/Home.vue:5]: Once working, look into adding it as a separate component so we can reuse it in the admin dashboard as well.

**Review by zaidi-01 — CHANGES_REQUESTED** [review]

File: client/vue.config.js (hunk 1 above)
Comment (zaidi-01) [reply client/vue.config.js:8]: The documentation is based around port 5000 being the default port.

**Comment by 4RHollingsworth** [comment]
I think I resolved the issues, will work on making the search bar component once we're happy with how it looks so far

**Review by zaidi-01 — APPROVED** [review]

**Pull request merged**